src/main/main.c

```diff
@@ -311,8 +311,14 @@ void init(void)
     spiInit(SPIDEV_1);
 #endif
 #ifdef USE_SPI_DEVICE_2
+#ifdef OMNIBUSF4
```

Comment:
It's a pity to add a target-specific `#ifdef` here, since we have tried to get rid of target-specific code. Is there any way it can be avoided?